examples/run_nlp_trainer.py

```diff
@@ -0,0 +1,365 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Entrypoint that runs the Composer trainer on a provided YAML hparams file.
+Specifically designed for the NLP use case, allowing pre-training and fine-tuning on 
+downstream tasks to be handled within one script.
+
+Example that pretrains a BERT:
+    >>> composer examples/run_nlp_trainer.py 
+    -f composer/yamls/models/bert-base.yaml 
+    --training_scheme pretrain 
+
+Example that pretrains and finetunes a BERT:
+    >>> composer examples/run_nlp_trainer.py 
+    -f composer/yamls/models/bert-base.yaml 
+    --training_scheme all 
+    --save_folder ~/checkpoints
+
+Example that finetunes a pretrained BERT:
+
+    >>> composer examples/run_nlp_trainer.py
+    -f composer/yamls/models/bert_base.yaml
+    --training_scheme finetune
+    --load_path ~/checkpoints
```

Comment:
Do we ever check if the `load_path` exists? I was able to run this command but it completed instantly since `~/checkpoints` does not exist.

Reply:
I didn't check for this case in the entrypoint but composer should already have some sort of validation on this arg? cc @ravi-mosaicml

Reply:
Is `autoresume=True`? If so then it'll start from scratch. Otherwise it should crash.